superbuild/projects_modules/quazip.cmake

```diff
@@ -6,7 +6,8 @@ set(ep quazip)
 ## List the dependencies of the project
 ## #############################################################################
```

Comment:
Is there a way to deactivate the test. I got errors  due to a strange path. with .qpro it was possible

Reply:
I don't understand, which test ? (line ?)